src/components/UsersTableComponent.js

```diff
@@ -72,44 +72,53 @@ const defaultSorted = [
 
 const mapStateToProps = (state) => {
   return {
-    users: state.users
-  }
-}
+    getUsersList: state.users.getUsersList,
```

Comment:
hmm I think if this is something that store data, you have to name it with `noun` instead of `verb`. User `verb` for function

Reply:
thanks for advice, i will fix it asap

Reply:
the problem is already fixed by upcoming commit